share/spice/rand_word/single_word.handlebars

```diff
@@ -0,0 +1,2 @@
+<h3 class="c-list__title">Random Word</h3>
```

Comment:
The title should still be handled by the Text template, not in the `content`.

Let's remove everything in here except the `<a>` tag